sql/information_schema/columns_table.go

```diff
@@ -167,14 +167,18 @@ func columnsRowIter(ctx *sql.Context, cat sql.Catalog, columnNameToDefault map[s
 					ordinalPos uint64
 					colDefault interface{}
 					charMaxLen interface{}
-					colType    string
+					columnKey  string
+					colType    = strings.ToLower(c.Type.String())
 				)
+
+				ordinalPos = uint64(i + 1)
 				if c.Nullable {
 					nullable = "YES"
 				} else {
 					nullable = "NO"
 				}
-				dataType = strings.ToLower(c.Type.String())
+
+				dataType = colType
```

Comment:
(minor) you might also try putting `dataType` and `ordinalPos` into the var block above. It doesn't make very much difference, but nice to initialize all these variables together since they are used all together at the end of the method to create the row.